specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2021-09-01-preview/appplatform.json

```diff
@@ -3695,7 +3707,62 @@
           },
           "readOnly": true
         }
-      }
+      },
+      "discriminator": "type"
+    },
+    "KeyVaultCertificateProperties": {
```

Comment:
To double confirm, API caller doesn't need to specify it is calling "KeyVaultCertificateProperties" or "ContentCertificateProperties", right?